### PR TITLE
Recover the  machine set for debugging on build01

### PR DESCRIPTION
**Pull request by hongkailiu**
Follow up https://coreos.slack.com/archives/C01MW0NE1S8/p1613060873126400?thread_ts=1613060834.126000&cid=C01MW0NE1S8

```bash
$ git checkout 00e17038f272690c423361eb8f6b4a5e282d66a5  ./clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a-debug_machine_set.yaml
```

/cc @openshift/openshift-team-developer-productivity-test-platform  @mrunalp

### CODE REVIEW

File: clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a-debug_machine_set.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        key: dptp-debug
+        value: "true"
+      providerSpec:
+        value:
+          ami:
+            id: ami-06f85a7940faa3217
+          apiVersion: awsproviderconfig.openshift.io/v1beta1
+          blockDevices:
+          - ebs:
+              volumeSize: 300
```
Comment (hongkailiu) [inline clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a-debug_machine_set.yaml:38]: This was the disk size from the last time.

Now for workers, it 1.5T:

https://github.com/openshift/release/blob/ec731d801150150c999429dd603175a24bc00d9d/clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a_machine_set.yaml#L31-L33

@mrunalp

What size do you want for the debugging node?

Comment (mrunalp) [reply clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a-debug_machine_set.yaml:38]: I want to be the same as the regular CI worker nodes.

Comment (hongkailiu) [reply clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a-debug_machine_set.yaml]: Done.

**Comment by umohnani8** [comment]
/hold
Will ping when we are ready to experiment with a debug node

**Comment by hongkailiu** [comment]
/close

debugging goes to another direction.

https://github.com/openshift/release/pull/17307

**Pull request closed without merging**